src/test/java/htsjdk/samtools/cram/io/DefaultBitStreamTest.java

```diff
@@ -0,0 +1,242 @@
+package htsjdk.samtools.cram.io;
+
+import htsjdk.HtsjdkTest;
+import htsjdk.samtools.util.RuntimeEOFException;
+import htsjdk.samtools.util.RuntimeIOException;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+// tests DefaultBitInputStream and DefaultBitOutputStream
```

Comment:
New file to address coverage in these 2.  Found and fixed one bug regarding `DefaultBitInputStream.reset()`